packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -289,9 +289,9 @@ export class AccordionItem implements ConditionalSlotComponent, LoadableComponen
       return;
     }
 
-    accordionItem.iconPosition = closestAccordionParent.iconPosition;
-    accordionItem.iconType = closestAccordionParent.iconType;
-    accordionItem.scale = closestAccordionParent.scale;
+    this.iconPosition = closestAccordionParent.iconPosition;
```

Comment:
while both are valid, in Lit, to get better typing inside the component you should be settings things on `this` rather than `this.el` to get better typing